salt/modules/vagrant.py

```diff
@@ -75,16 +73,11 @@ def _vagrant_ssh_config(vm_):
     '''
     machine = vm_['machine']
     log.info('requesting vagrant ssh-config for VM %s', machine or '(default)')
-    cmd = _runas_sudo(vm_, 'vagrant ssh-config {}'.format(machine))
-    try:
-        output = subprocess.check_output(
-            [cmd],
-            shell=True,
-            cwd=vm_.get('cwd', None)
-            )
-    except subprocess.CalledProcessError as e:
-        output = e.output  # if the return code was non-zero, use the output anyway
-    reply = salt.utils.stringutils.to_str(output)
+    cmd = 'vagrant ssh-config {}'.format(machine)
```

Comment:
@cachedout does this need to be `{0}` still for following the old standards, or can we stop doing that now that we have dropped py2.6?